test/byContract/IexecEscrow/IexecEscrowToken.test.ts

```diff
@@ -0,0 +1,473 @@
+                .withArgs(iexecPoco, accountA.address, standardAmount)
+                .to.changeTokenBalances(iexecPoco, [accountA], [-standardAmount])
+                .to.emit(iexecPoco, 'Transfer')
+                .withArgs(accountA.address, AddressZero, standardAmount);
+        });
+        it('Should withdraw zero token', async () => {
+            expect(await iexecPocoAsAccountA.callStatic.withdraw(0)).to.be.true;
+            await expect(iexecPocoAsAccountA.withdraw(0))
+                .to.changeTokenBalances(rlcInstance, [iexecPoco, accountA], [-0, 0])
+                .to.emit(rlcInstance, 'Transfer')
+                .withArgs(iexecPoco, accountA.address, 0)
+                .to.changeTokenBalances(iexecPoco, [accountA], [-0])
+                .to.emit(iexecPoco, 'Transfer')
+                .withArgs(accountA.address, AddressZero, 0);
+        });
+        it('Should not withdraw native tokens with empty balance', async () => {
+            await expect(
+                iexecPocoAsAccountA.withdraw(standardAmount),
+            ).to.be.revertedWithoutReason();
+        });
+        it('Should not withdraw tokens with insufficient balance', async () => {
+            await rlcInstanceAsAccountA
+                .approve(iexecPoco.address, standardAmount)
+                .then((tx) => tx.wait());
+            await iexecPocoAsAccountA.deposit(standardAmount).then((tx) => tx.wait());
+
+            await expect(
+                iexecPocoAsAccountA.withdraw(standardAmount.mul(2)),
+            ).to.be.revertedWithoutReason();
+        });
+    });
+
+    describe('Withdraw to', () => {
+        it('Should withdraw another address', async () => {
+            await rlcInstanceAsAccountA
+                .approve(iexecPoco.address, standardAmount)
+                .then((tx) => tx.wait());
+            await iexecPocoAsAccountA.deposit(standardAmount).then((tx) => tx.wait());
+
+            const withdrawToParams = {
+                amount: standardAmount,
+                target: accountB.address,
+            };
+            const withdrawToArgs = Object.values(withdrawToParams) as [BigNumber, string];
+
+            expect(await iexecPocoAsAccountA.callStatic.withdrawTo(...withdrawToArgs)).to.be.true;
+            await expect(iexecPocoAsAccountA.withdrawTo(...withdrawToArgs))
+                .to.changeTokenBalances(
+                    rlcInstance,
+                    [iexecPoco, withdrawToParams.target],
+                    [-withdrawToParams.amount, withdrawToParams.amount],
+                )
+                .to.emit(rlcInstance, 'Transfer')
+                .withArgs(iexecPoco, withdrawToParams.target, withdrawToParams.amount)
+                .to.changeTokenBalances(iexecPoco, [accountA], [-standardAmount])
+                .to.emit(iexecPoco, 'Transfer')
+                .withArgs(accountA.address, AddressZero, withdrawToParams.amount);
+        });
+        it('Should withdraw To with zero token', async () => {
```

Comment:
```suggestion
        it('Should withdraw to another address with zero token', async () => {
```
(multiples occurrences)

Reply:
[several cleans](https://github.com/iExecBlockchainComputing/PoCo/pull/141/commits/4f91a47f4087fb404718953064ef188b393de48d)